.travis.yml

```diff
@@ -28,38 +28,22 @@ env:
 addons:
   apt:
     packages:
-    - libcurl4-openssl-dev
-    - python-numpy
-    - python-opencv
-    - libimage-exiftool-perl
-    - libopencv-dev
     - libjpeg-progs
-    - libjpeg-dev
-    - libpng-dev
-    - libx264-dev
-    - libass-dev
-    - libvpx1
-    - libvpx-dev
-    - libwebp-dev
-    - webp
+    - libimage-exiftool-perl
     - gifsicle
-    - python-numpy
-    - python-scipy
-    - libcairo2-dev
-    - gfortran
-    - libopenblas-dev
-    - liblapack-dev
-    - python-pyexiv2
-    - yasm
+    - python-all-dev
+    - scons
+    - libboost-python-dev
+    - libexiv2-dev
 before_install:
   - echo "deb http://archive.ubuntu.com/ubuntu zesty main universe" | sudo tee --append /etc/apt/sources.list
   - sudo apt-get update
   - sudo apt-get install ffmpeg -y -f
 install:
-  - pip install -U --upgrade pip
+  - pip install --upgrade pip
+  - pip install -I https://github.com/escaped/pyexiv2/archive/master.zip
```

Comment:
Hi @kkopachev, `master` branch may not be completely stable at a given moment and could break Thumbor build. What do you think about installing this dependency via .deb package?

Reply:
not sure I can use deb to make it automatically available in virtual env. I can reference commit hash instead.

Reply:
pyexiv2 is dead anyway, not likely there will be any releases.

Reply:
Currently, pyexiv2 is installed via [.deb package](https://github.com/thumbor/thumbor/blob/master/.travis.yml#L52) (Travis) and the [tests are running]( https://travis-ci.org/thumbor/thumbor/jobs/274855100#L2641).

Reply:
current travis setup uses python with system provided packages, that's why it's working.
However, not working if we don't use system python and use virtualenv: https://travis-ci.org/thumbor/thumbor/jobs/280505386#L1689

Reply:
Also, if you guys care that much about something breaking the build, there are other issues as well https://github.com/thumbor/thumbor/pull/975/files#r141397089